generator/source/faq/validator.rst

```diff
@@ -248,8 +248,12 @@ What does this error mean: ``sawtooth-validator[... ERROR cli] Cannot have a gen
 ------------------------------------------------------------------------------------------------------------------------
 You tried to create a new genesis block when you did not need to (because there already is a genesis block). To solve, this remove file ``/var/lib/sawtooth/genesis.batch.file`` and restart ``sawtooth-validator`` .
 
-I get this error when testing with a lot of validators: ``Max occupancy was not provided by transaction processor: ... Using default max occupancy: 10``
+I get this warning when running a transaction processor: ``Max occupancy was not provided by transaction processor: ... Using default max occupancy: 10``
```

Comment:
If I wrap it to 80 character, it is not picking up this line as a single question (Question list showing up on top). Only the last line with the underline is considered as a question. How to go about it?